sw/device/silicon_creator/mask_rom/e2e_tests/mask_rom_e2e_testplan.hjson

```diff
@@ -0,0 +1,360 @@
+// Copyright lowRISC contributors.
+// Licensed under the Apache License, Version 2.0, see LICENSE for details.
+// SPDX-License-Identifier: Apache-2.0
+{
+  name: "mask_rom_e2e"
+
+  testpoints: [
+    // Bootup without ROM_EXT
+    {
+      name: mask_rom_e2e_bootup_no_rom_ext
+      desc: '''Verify device fails to boot with no ROM_EXT.
+            
+            - Test runner loads the Mask ROM and no ROM_EXT.
+            - Attempt to boot.
+            - Verify that the chip indicates the correct boot fault.
+            '''
+      target: ["verilator", "dv_sim", "fpga", "silicon"]
+      tests: []
+    }
+    // Bootup with ROM_EXT
+    {
+      name: mask_rom_e2e_bootup_success
+      desc: '''Verify device boots with a good ROM_EXT.
+
+            - Generate a ROM_EXT with a sufficiently high version number.
```

Comment:
We should test both A/B partitions, across valid life cycles.  Should we also add a separate test for otbn versus sw sigverify?

Reply:
Do you think we should perform all tests on both A and B partitions, or is just performing this test on both sufficient (and we run the rest on just the A partition)?